suite-native/transactions/src/components/TransactionsList/TransactionIconSpinner.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from 'react';
+import Animated, {
+    useAnimatedStyle,
+    useSharedValue,
+    withRepeat,
+    withTiming,
+    cancelAnimation,
+    Easing,
+} from 'react-native-reanimated';
+
+import { Canvas, Circle, vec, SweepGradient } from '@shopify/react-native-skia';
+
+import { Box } from '@suite-native/atoms';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+type TransactionIconSpinnerProps = {
+    radius: number;
+    color: Color;
+};
+
+const STROKE_WIDTH = 8;
+
+const ContainerStyle = prepareNativeStyle(_ => ({
+    position: 'absolute',
+}));
+
+export const TransactionIconSpinner = ({ radius, color }: TransactionIconSpinnerProps) => {
+    const { applyStyle, utils } = useNativeStyles();
+
+    const rotation = useSharedValue(0);
+    const animatedStyles = useAnimatedStyle(
+        () => ({
+            transform: [
+                {
+                    rotateZ: `${rotation.value}deg`,
+                },
+            ],
+        }),
+        [rotation.value],
+    );
+
+    useEffect(() => {
+        rotation.value = withRepeat(
+            withTiming(360, {
+                duration: 2500,
+                easing: Easing.linear,
+            }),
+            -1,
```

Comment:
nit: Please use constants as in other files. At least for the `-1` value so it's readable from const name what it is without having to think about it too much

Reply:
rename in ec7bdb40a